lib/get-selectors.js

```diff
@@ -8,6 +8,12 @@ function isNestedRule(node) {
   return /(?:at)?rule/.test(node.parent.type)
 }
 
+function isOnlyExtends(node) {
+  return node.nodes && node.nodes.length === 1 && node.every(function(child) {
+    return child.type === 'atrule' && child.name === 'extend'
+  });
+}
+
```

Comment:
Do we need to think about more than one extend here? Like:

```css
.selector {
  @extend .test;
  @extend .blah;
}
```

Perhaps `hasNoDeclarations` needs to be repurposed/renamed to handle extra cases so that we only need to iterate with `every` once.

What do you think?